src/auth/Gadgets.cc

```diff
@@ -129,10 +129,6 @@ authenticateCachedUsersList()
 #if HAVE_AUTH_MODULE_NEGOTIATE
     if (Auth::SchemeConfig::Find("negotiate"))
         u1 = Auth::Negotiate::User::Cache()->sortedUsersList();
-#endif
-#if HAVE_AUTH_MODULE_NTLM
-    if (Auth::SchemeConfig::Find("ntlm"))
-        u2 = Auth::Ntlm::User::Cache()->sortedUsersList();
 #endif
```

Comment:
Why are we no longer reporting NTLM users? The PR description is silent about this seemingly out of scope change.